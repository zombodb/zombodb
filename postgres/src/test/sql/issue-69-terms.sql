SELECT *
FROM zdb_tally('so_posts', 'title', '^a.*', '', 5000, 'term')
LIMIT 10;
