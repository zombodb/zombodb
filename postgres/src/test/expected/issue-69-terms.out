SELECT *
FROM zdb_tally('so_posts', 'title', '^a.*', '', 5000, 'term')
LIMIT 10;
   term   | count  
----------+--------
          | 129680
 a        |   9898
 a.append |      1
 a.i      |      3
 a7       |      1
 aa       |      1
 aac      |      1
 aardvark |      1
 ab       |      1
 abalone  |      1
(10 rows)

